win32ss/drivers/videoprt/videoprt.c

```diff
@@ -294,6 +296,8 @@ IntVideoPortFindAdapter(
                                       SystemBasicInfo.PageSize;
     }
 
+    // FIXME: Check the adapter key and update VideoDebugLevel variable.
```

Comment:
It would be nice to fix this thing in this PR :) Probably using `RtlQueryRegistryValues` function.